node/cli/src/cli.rs

```diff
@@ -235,55 +248,3 @@ impl RelayChainCli {
 		}
 	}
 }
-
-/// Block authoring scheme to be used by the dev service.
-#[derive(Debug)]
-pub enum Sealing {
-	/// Author a block immediately upon receiving a transaction into the transaction pool
-	Instant,
-	/// Author a block upon receiving an RPC command
-	Manual,
-	/// Author blocks at a regular interval specified in milliseconds
-	Interval(u64),
```

Comment:
Can we use more `git mv` to preserve the history of code blocs that have moved? If it's not too hard I'd like to preserve history. If it's gonna be a pain, the we can just leave it.